### ISSUE TITLE
Why didn't I find this ages ago? :D

### ISSUE THREAD

**Issue by billiegoose** [data-ux/wasm-playground#2]
I just spent the better part of 2 days struggling to create something just like this [(try-wasm)](https://github.com/wmhilton/try-wasm). Is this a project that you'd feel comfortable moving to the [WebAssembly Github group](https://github.com/WebAssembly/) where it would be more visible and more people would discover it?

**Comment by data-ux** [comment]
@wmhilton Yes, we can move it into the WebAssembly group. How should we proceed?

**Comment by billiegoose** [comment]
Yo @WebAssembly! This thing is cool, you guys should signal boost it ❤️
